File: 13-Python/hello_world/conditionals.py
import platform


def get_current_os():
    platform_system = platform.system()
    if platform_system == "Linux":
        return "Superior OS"
    elif platform_system == "Windows":
        return "Inferior OS"
    else:
        return "Mediocre OS"


print(get_current_os())

File: 08-Asinchroninis programavimas/language-examples/conditionals.js
function getCurrentOS() {
    const system = process.platform;
    let result;
    switch (system) {
        case "linux":
            result = "Superior OS";
            break;
        case "win32":
            result = "Inferior OS";
            break;
        default:
            result = "Mediocre OS";
    }
    return result;
}

console.log(getCurrentOS());